string topics
string timestamp
---
bool suc
string message